requirements/requirements_authentication.in

```diff
@@ -13,7 +13,7 @@ ldap-filter
 python3-saml
 tacacs_plus
 
-xmlsec==1.3.13 # Pin for https://github.com/xmlsec/python-xmlsec/issues/314
+xmlsec==1.3.17
```

Comment:
Can we add a comment to why we are moving this pin to 1.3.17 instead of just updating to the latest?